packages/experiments/src/components/Shimmer/ShimmerElementsGroup/ShimmerElementsGroup.types.ts

```diff
@@ -0,0 +1,59 @@
+import * as React from 'react';
+import { IStyle, ITheme } from '../../../Styling';
+import { IStyleFunction } from '../../../Utilities';
+import { IShimmerElement } from '../Shimmer.types';
+
+export interface IShimmerElementsGroup {
+}
+
+/**
+ * ShimmerElementsGroup component props.
+ */
+export interface IShimmerElementsGroupProps extends React.AllHTMLAttributes<HTMLElement> {
+  /**
+   * Optional callback to access the IShimmerElementsGroup interface. Use this instead of ref for accessing
+   * the public methods and properties of the component.
+   */
+  componentRef?: (component: IShimmerElementsGroup | null) => void;
+
+  /**
+   * Optional maximum row height of the shimmerElements container.
+   */
+  rowHeight?: number;
+
+  /**
+   * Elements to render in one group of the Shimmer.
+   */
+  shimmerElements?: IShimmerElement[];
+
+  /**
+   * Optional boolean for enabling flexWrap of the container containing the shimmerElements.
+   * @default false
+   */
+  flexWrap?: boolean;
```

Comment:
Should this be `true` by default?

Reply:
No. This subcomponent is also used internally by ShimmerBase when everything stays in one line. So by default is false.